composer.json

```diff
@@ -29,5 +29,10 @@
         "psr-4": {
             "OpenTracingTests\\": "./tests/"
         }
+    },
+    "scripts": {
+        "test": "php ./vendor/bin/phpunit tests",
+        "check-linting": "./vendor/bin/phpcs --standard=ZEND --standard=PSR2 --ignore=*/vendor/* ./",
```

Comment:
me neither. `lint` would be imperative like `test`. `fix-lint` is not 100% correct, but I would prefer a short command over perfect grammar. I would read it as "fix the lint task".